packages/features/bookings/Booker/utils/use-brand-colors.tsx

```diff
@@ -17,10 +15,7 @@ export const useBrandColors = ({
     lightVal: brandColor,
     darkVal: darkBrandColor,
   });
-  useCalcomTheme(brandTheme);
-  const { setTheme } = useTheme(theme);
 
-  useEffect(() => {
-    if (theme) setTheme(theme);
```

Comment:
We shouldn't set theme here. useTheme takes care of setting the theme automatically considering embed preferences.